pep-0394.txt

```diff
@@ -175,14 +236,13 @@ making such a change.
   versa. That way, if a sysadmin does decide to replace the installed
   ``python`` file, they can do so without inadvertently deleting the
   previously installed binary.
-* If the Python 2 interpreter becomes uncommon, scripts should nevertheless
+* When the Python 2 interpreter becomes uncommon, scripts should nevertheless
```

Comment:
We should reword this bullet point, since we now expect Python 3 -> Python 4 to be a "enough changes have accumulated since Python 3.0 that they're effectively different languages" kind of event, rather than a Python 3.0 style compatibility break. Something like:

> Even as the Python 2 interpreter becomes less common, scripts should nevertheless continue to use the `python3` convention rather than just `python`. In the event of a future Python 4.0 release, distributors may then choose to simply alias `python3` to the same binary as `python4` (since the degree of change is now expected to be the same as in any other Python feature release), or else they may choose to maintain parallel `python3` and `python4` stacks for a time (depending on the needs of their specific audience).

Reply:
I'd happy to change this, however is there some authoritative source I can link to? I don't want this PEP to be the place that promises/defines/guesses the plans for Python 4. If not, we could also reword this to work either way. Something like: "When Python 2 is forgotten, still use "python3". If Python 4 will be a disaster, it will help you, OTOH if Python 4 will be peace of cake, you can either alias "python3" to "python4" or decide to keep them parallel installable for a time."

Reply:
There is no authoritative statement beyond some email threads where that seems to be the prevailing feeling towards Python 4.

Reply:
That's my understanding as well. I don't want this PEP to be used as a reference about Python 4. I'll try to go with a statement that is more universal.

Reply:
36e61fab